autoload/neomake.vim

```diff
@@ -115,16 +115,16 @@ function! neomake#GetMaker(name, ...) abort
     endif
     let maker = copy(maker)
     let defaults = {
-        \ 'exe': a:name,
+        \ 'exe': a:maker_id,
         \ 'args': [],
-        \ 'errorformat': '',
```

Comment:
I don't think this change is right. Instead, in `s:ProcessJobOutput` there is a line `if has_key(a:maker, 'errorformat')` and then it sets up the errorformat in the `if` body. I think we should just change that to `if len(a:maker.errorformat)` and the behavior should then be the same as before my change.